src/index.html

```diff
@@ -5,9 +5,50 @@
     <meta name="viewport"
           content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
     <title>Stars</title>
-    <link rel="stylesheet" href="./style.css">
+    <link rel="stylesheet" href="styles/style.css">
   </head>
   <body>
-    <h1>Stars</h1>
+    <div class="stars--0">
+      <div class="stars__star"></div>
+      <div class="stars__star"></div>
+      <div class="stars__star"></div>
+      <div class="stars__star"></div>
+      <div class="stars__star"></div>
+    </div>
+    <div class="stars--1">
```

Comment:
![image](https://github.com/mate-academy/layout_stars/assets/54936044/bffe51f5-bc5f-4128-9b44-ae7207de0cac)

```suggestion
     </div>

    <div class="stars--1">
```